junit-test/shared/src/test/scala/org/scalajs/junit/utils/JUnitTest.scala

```diff
@@ -255,17 +262,19 @@ object JUnitTest {
         val rest: Array[String] = tail.mkString("").split(separator, -1)
         val testName: String = rest(0)
         val throwableToString: String = rest(1)
+        val durationPopulated: Boolean = rest(2).toBoolean
         Event(
           status = Status.values()(s - '0'),
           testName,
-          throwableToString = if (throwableToString.isEmpty) None else Some(throwableToString)
+          throwableToString = if (throwableToString.isEmpty) None else Some(throwableToString),
+          durationPopulated
         )
     }
 
     def serialize(o: Output): String = o match {
       case Log(level, msg) => "l" + level + msg
       case Done(msg)       => "d" + msg
-      case Event(s, n, t)  => "e" + s.ordinal + n + separator + t.getOrElse("")
+      case Event(s, n, t, d)  => "e" + s.ordinal + n + separator + t.getOrElse("") + separator + d
```

Comment:
Please keep the arrows aligned:
```suggestion
      case Log(level, msg)   => "l" + level + msg
      case Done(msg)         => "d" + msg
      case Event(s, n, t)    => "e" + s.ordinal + n + separator + t.getOrElse("")
      case Event(s, n, t, d) => "e" + s.ordinal + n + separator + t.getOrElse("") + separator + d
```

Reply:
done